apps/designer/app/designer/shared/fonts-manager/item-menu.tsx

```diff
@@ -42,6 +44,13 @@ const ItemMenu = ({
     };
   }, []);
 
+  const [authPermit] = useAuthPermit();
+
+  const deleteDisabled = authPermit === "view";
+  const tooltipContent = deleteDisabled
+    ? "View mode. You can't delete assets."
```

Comment:
probably @taylornowotny

Reply:
sorry meant to say we should let someone like @taylornowotny with native English check the wording